### PR TITLE
Selected Stories Block: Enable views and show block content in frontend

### LINKED ISSUE #45: Cover Images Not Displayed
Cover images are not being displayed in the following instances

- [x] While selecting stories
- [x] After selecting stories > enabling the `Show cover image`

Screenshot 2020-11-13 at 3 06 04 AM

### LINKED ISSUE #46: Selected Stories Block -  Not Displaying Content
On demo site, inspite of selecting stories to be displayed, on FrontEnd the stories are not being displayed (either on preview or once updated)

Screenshot 2020-11-13 at 3 13 41 AM

### LINKED ISSUE #47: Selected Stories Block - Enable Remaining Views
The Selected Web Stories block should have the below views supported similar to the Latest Web Stories block

- [x] List view
- [x] Grid view
- [x] Circle view
- [x] Carousel view

### PR DESCRIPTION
**Pull request by ediamin**
This PR:
- Fixes #45 
- Fixes #46 
- Closes #47

### CODE REVIEW

File: assets/src/dashboard/app/api/useStoryApi.js [hunk 1]
```diff
@@ -80,6 +81,7 @@ const useStoryApi = (dataAdapter, { editStoryURL, storyApi }) => {
         context: 'edit',
         _web_stories_envelope: true,
         search: searchTerm || undefined,
+        author: author || undefined,
```
Comment (divyarajmasani) [inline assets/src/dashboard/app/api/useStoryApi.js:84]: I believe this file is part of the Story editor, if so we need to check the test cases are working fine for that.

File: assets/src/selected-stories-block/block/edit.css [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+  font-size: 15px;
+  line-height: 22px;
+  font-weight: 700;
+  display: box;
+  -webkit-line-clamp: 2; /* Browser support: https://caniuse.com/?search=line-clamp */
+  box-orient: vertical; /* stylelint-disable-line property-no-unknown */
+  overflow: hidden;
+}
+
+.wp-block-web-stories-selected-stories.is-view-type-list
```
Comment (divyarajmasani) [inline assets/src/selected-stories-block/block/edit.css:37]: Can we not use the `latest-web-stories` style here, we may not need this entire file. Maybe add `selected-stories` as a separate class for identification.

File: assets/src/dashboard/app/api/useStoryApi.js (hunk 1 above)
Comment (ediamin) [reply assets/src/dashboard/app/api/useStoryApi.js:84]: @divyarajmasani In the automated check list, both e2e and js unit tests are passed. So I guess it's OK.

File: assets/src/selected-stories-block/block/edit.css (hunk 2 above)
Comment (ediamin) [reply assets/src/selected-stories-block/block/edit.css:37]: If we can use less/scss, then it'll be very easy to re use the style sheet. But I'm not sure if we can directly use the entire css from latest web stories. Need to check that out.

**Review by divyarajmasani — APPROVED** [review]

**Pull request merged**